api/v1/server.gotmpl

```diff
@@ -122,6 +123,28 @@ var (
   tlsCACertificate  string
 )
 
+var SpecCell = cell.Module(
+	"{{ dasherize .Name }}-spec",
+	"{{ humanize .Name }} Specification",
+
+	cell.Provide(newSpec),
+)
+
+type Spec struct {
+	*loads.Document
+}
+
+func newSpec() (*Spec, error) {
```

Comment:
Please rebase and change the existing `Cell` to also provide the spec, e.g. `newForCell` could return `(*Spec, *Server, error)` and remove `SpecCell`.

Reply:
Ah, after reading rest of the commits I realized we'll be using this in situations where we're not yet using the `server.Cell`. Perhaps then we'll want to change `newForCell` to depend on `*Spec` instead. At least for the time being.

Reply:
Ah yes I missed that during the rebase, good point. I'll fix that up.

Reply:
Diff:
```
@@ -57,11 +58,7 @@ type serverParams struct {                                                                                                                                                                                                                                              
 }                                                                                                                                                                                                                                                                                         
                                                                                                                                                                                                                                                                                           
 func newForCell(p serverParams) (*Server, error) {                                                                                                                                                                                                                                        
-       swaggerSpec, err := loads.Analyzed(SwaggerJSON, "")                                                                                                                                                                                                                                
-       if err != nil {                                                                                                                                                                                                                                                                    
-               return nil, fmt.Errorf("failed to swagger spec: %w", err)                                                                                                                                                                                                                  
-       }                                                                                                                                     
-       api := {{ .Package }}.New{{pascalize .Name}}API(swaggerSpec)                                                                          
+       api := {{ .Package }}.New{{pascalize .Name}}API(p.Spec)                                                                               
                                                                                                                                              
        // Construct the API from the provided handlers                                                                                       
        {{- $package := .Package }} 
```